### DISCUSSION TITLE
Problemi switch sonoff dopo ultimo agg.to HA

### DISCUSSION THREAD

**Discussion by Borghese** [indomus/forum#2819] [GENERALE - Discussioni generiche]
ciao a tutti,
ieri ho aggiornato HA alla versione 2024.1.1 e da allora ho un problema a tutte le card con cui comando i punti luce, tramite dei sonoff mini wifi.
Mi da solo la possibilità o di accendere o di spegnere, e non si aggiorna poi lo stato acceso/spento
Questo problema ce l'ho sia con le card entity di mushroom, che con la classica interrutore originale di HA
Il bello è che agendo invece direttamente sull'entità, funziona tutto a dovere (così come da alexa o ewelink)

Avete idee ? E capitato solo a me ?!
oggi ho aggiornato alla 2024.1.2 ma non ha risolto

Lo imputo all'aggiornamento perchè non ho fatto altre modifiche nel frattempo, quella è l'unica variabile.
Al netto dell'installazione di ZHA per integrare un paio di sensori aqara, ma sta su canale 25 lo zigbee, non credo sia un errore di comunicazione che dite ?

**Comment by KC-inDomus** [comment]
con cosa integri i sonoff?

**Reply by Borghese** [reply]
I sonoff in ha sono integrati con loro add on ufficiale. 
ma hanno sempre funzionato, non capisco....
Ho provato a disabilitare qualche entità e riabilitarla, ricaricare l'add on, ricreare la card... nulla! 
Ma può essere che disturbi Zigbee ?!?

**Comment by Borghese** [comment]
Niente da fare, le ho provate praticamente tutte :

Ho rimosso l'add on di zigbee (ZHA) per capire se era lui che dava fastidio (difficile perchè è sul canale 25, bello distante dal canale wifi, ma cmq anche togliendolo non ho risolto)

Ho rimosso l'add on di sonoff, e reintegrato, problema persiste

Ho cancellato la card lovelace, e rifatta, sia usando le card switch che entità originali di HA, sia quelle entity di mushroom, niente! 

La cosa assurda è che : 

Se agisco dall'entità dentro l'integrazione, funziona tutto alla perfezione, accende spegne e l'icona si illumina correttamente in base allo stato, per intenderci questa : 

<a href="https://imgbb.com/"><img src="https://i.ibb.co/7JnStfK/aaa.png" alt="aaa" border="0"></a>

se invece uso la card entità di HA non funziona, non commuta alcun input, e addirittura se switcho per accendere, dopo un secondo ritorna da sola in posizione spenta, per intenderci questa : 

<a href="https://imgbb.com/"><img src="https://i.ibb.co/qj815nk/bbb.png" alt="bbb" border="0"></a>

Usando la entity card di mushroom invece, non fa proprio nulla, è morta (ovviamente funzione tocco è su commuta)

<a href="https://imgbb.com/"><img src="https://i.ibb.co/3dwZxwX/ccc.png" alt="ccc" border="0"></a>

Non so piu che fare o che pensare
Ultima alternativa sarebbe downgradare ha ad aggiornamento precedente, ma onestamente non ho voglia, e spero si risolva in update successivo

ma possibile abbia riscontrato solo io questo problema ? 

C'è qualche log che posso guardare per capire cosa succede ?

**Reply by Scialla** [reply]
se le integri con sonoffLAN c'era un problema sull'integrazione che però, ora, hanno sistemato. Non so se l'eventuale integrazione che usi possa avere un problema analogo e debba ancora essere sistemata. Prova a guardare le issue su github

**Reply by Borghese** [reply]
Bingo! su github ho trovato altri utenti con steso problema dopo ultimo aggiornamento : https://github.com/AlexxIT/SonoffLAN/issues/1311

Dicono che si risolve con la versione 3.5.4 dell'integrazione Sonoff, ma a me ancora non la propone, e sono fermo alla  3.5.3 :

    "sonoff": {
      "version": "3.5.3",
      "requirements": [
        "pycryptodome>=3.6.6"

Aspetterò....

**Reply by Scialla** [reply]
La 3.5.4 è stata rilasciata 4 giorni fà, e infatti è risolutiva.
Io, comunque, avevo già risolto in precedenza modificando a mano il file così come consigliavano nel topic principale, qui:
https://github.com/AlexxIT/SonoffLAN/issues/1288#issuecomment-1876688902

**Reply by Borghese** [reply]
Grazie Scialla,
ma solo per capire io : Se vado sull'integrazione installata, opzione "scarica diagnostica" vedo che io ho installato la 3.5.3
Se la 3.5.4 è stata rilasciata qualche gg fa, secondo te come mai il mio HA non me la propone tra gli aggiornamenti ?
Non trovo neanche un opzione per forzare l'aggiornamento di questo singolo componente.... cosa mi sto perdendo ?

**Reply by Borghese** [reply]
come non detto sono un pirla io! 
L'integrazione non è ufficiale, e l'agg.to era proposto dentor hacs... io lo cercavo su quelli ufficiali di HA e me lo sono perso
Fatto tutto risolto!! Grazie 1000

p.s ovviamente funzionava anche la risoluzione manuale commentando la riga40 da file editor...
